src/endpoints/routing.cc

```diff
@@ -544,6 +544,23 @@ api::plan_response routing::operator()(boost::urls::url_view const& url) const {
     utl::verify(tt_ != nullptr && tags_ != nullptr,
                 "mode=TRANSIT requires timetable to be loaded");
 
+    auto const max_results = config_.timetable_
+                                 .and_then([](config::timetable const& x) {
+                                   return std::optional{x.plan_max_results_};
+                                 })
+                                 .value_or(256U);
+    utl::verify(query.numItineraries_ <= max_results,
+                "maximum number of minimum itineraries is {}", max_results);
+    auto const max_timeout = std::chrono::seconds{
+        config_.timetable_
+            .and_then([](config::timetable const& x) {
```

Comment:
`utl::verify(tt_ != nullptr, ...)` implies that config_.timetable_ has to be set too.
If in doubt just add it to the `utl::verify` statement above and assume that it's valid here.